### PR TITLE
fix: dependency and vulnerability for docker files

### PR DESCRIPTION
**Pull request by ihaardik**
Fixed the dependency issue of Dockerfile for past commit which failed to create docker image.
past commit: https://github.com/Countly/countly-server/pull/3578/files

### CODE REVIEW

**Comment by kaanklky** [comment]
Waiting for @ihaardik to test all Enterprise Docker images too.

**Review by kaanklky — COMMENTED** [review]
Checked all commits & Hardik already tested changes for all EE builds.

File: Dockerfile-api [hunk 1]
```diff
@@ -39,7 +57,7 @@ RUN curl -s -L -o /tmp/tini.deb "https://github.com/krallin/tini/releases/downlo
 	# cleanup & chown
 	npm remove -y --no-save mocha nyc should supertest && \
 	apt-get remove -y git gcc g++ make automake autoconf libtool pkg-config unzip sqlite3 && \
-	apt-get -y install libgbm-dev libgbm1 gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils && \
```
Comment (ar2rsawseen) [inline Dockerfile-api:42]: we need those for puppeteer I think

File: Dockerfile-api [hunk 2]
```diff
@@ -39,7 +57,7 @@ RUN curl -s -L -o /tmp/tini.deb "https://github.com/krallin/tini/releases/downlo
...
+	apt-get -y install gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils && \
```
Comment (ar2rsawseen) [inline Dockerfile-api:60]: We need this for puppeteer I think

**Pull request merged**